backend/src/domain/Games.js

```diff
@@ -0,0 +1,44 @@
+import {Game} from './models/Game';
+
+/**
+  * This class stores a list of all active games in session
+  */
+export class Games {
+  /**
+    * Initiates map storing all game sessions
+    */
+  constructor() {
+    this.activeGames = new Map();
```

Comment:
would an array be better here? since it'll be easier to interate through.... not sure how often we need to though.

Reply:
Map would be faster with larger number of game rooms, plus you can still iterate through a map